HW-3/task1.js

```diff
@@ -50,7 +50,7 @@ item_6=15;
 item_6_type=typeof item_6;
 //console.log("item_6==" + item_6, "item_6_type==" + item_6_type);
 let item_7;
-item_7=typeof typeof item_6;
+item_7="item_6";
```

Comment:
In this line you assigned a string `"item_6"` to variable `item_7`, but the task asks you to convert variable `item_6` to String

Reply:
Allready done.